requirements/check-avail-strategies.py

```diff
@@ -0,0 +1,7 @@
+import bagua  # noqa: F401
+import deepspeed  # noqa: F401
+import fairscale  # noqa: F401
+import horovod.torch
+
+# returns an error code
+assert horovod.torch.nccl_built()
```

Comment:
This assertion is failing and blocking master:

```python
Extension horovod.torch has not been built: /usr/local/lib/python3.7/dist-packages/horovod/torch/mpi_lib/_mpi_lib.cpython-37m-x86_64-linux-gnu.so not found
If this is not expected, reinstall Horovod with HOROVOD_WITH_PYTORCH=1 to debug the build error.
Warning! MPI libs are missing, but python applications are still available.
Traceback (most recent call last):
  File "requirements/check-avail-strategies.py", line 7, in <module>
    assert horovod.torch.nccl_built()
AttributeError: module 'horovod.torch' has no attribute 'nccl_built'
```

Reply:
ok, so it seems it is working as expected :)
what kind of jobs is failing?